tox.ini

```diff
@@ -50,6 +50,15 @@ usedevelop = true
 deps =
     -r{toxinidir}/requirements.txt
 
+# This is used for Windows, since binary name is different
+[testenv:mkvenv-win]
+basepython = python.exe
```

Comment:
Will this work when the Python 2.7 `.exe` is in `%PATH%`? What about using `python3.5` as basepython:

> On Windows a pythonX.Y named executable will be searched in typical default locations using the C:\PythonX.Y\python.exe pattern.

(from [the tox docs](https://testrun.org/tox/latest/example/general.html#basepython-defaults-overriding)). Then we could be sure that we'll be using Python 3 (but we'd probably have the user change the line to `python3.4` if they only have 3.4 installed, or if they don't use the default location).